test/integration/apiserver/admissionwebhook/timeout_test.go

```diff
@@ -303,11 +303,21 @@ func testWebhookTimeout(t *testing.T, watchCache bool) {
 				if err == nil {
 					t.Fatalf("expected error but got none")
 				}
+
+				expected := false
 				if tt.errorContains != "" {
-					if !strings.Contains(err.Error(), tt.errorContains) {
-						t.Errorf("expected an error saying %q, but got: %v", tt.errorContains, err)
+					if strings.Contains(err.Error(), "stream error"){
+						// refer to https://github.com/kubernetes/kubernetes/issues/98606#issuecomment-774832633
+						// for the reason for triggering this scenario
+						expected = true
```

Comment:
not all use cases in this test warrant a `stream error`, how about changing `errorContains` to a slice of string? 
```
errorContainsAnyOf: []string{
  "the server was unable to return a response in the time allotted",
  "stream error"
}
```

Reply:
ok~. In response to your suggestion, I have made changes. PTAL